code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -202,8 +202,12 @@
 
 			for(var/datum/limb/org in limbs)
 				var/status = ""
+				var/treat = ""
 				var/brutedamage = org.brute_dam
 				var/burndamage = org.burn_dam
+				var/brute_treated = org.is_bandaged()
+				var/burn_treated = org.salve() //org.is_salved() doesn't seem to work here.
```

Comment:
This means you are salving the would by examining it.
`is_salved()` seems to have for some reason `var/rval = 1` defined when it supposedly should be `var/rval = FALSE` instead, like `is_bandaged()` and `is_disinfected()`.
Test if it's not actually telling you the contrary of what you want, that is salved when it's not, and vice-versa.
If that's the case, just make that change for this to work.
That would be a pretty embarrassing bug.